packages/react/src/components/Tabs/Tabs-story.scss

```diff
@@ -1,6 +1,10 @@
 // IMPORTANT: This import path should _not_ be used outside our source tree
 // as `src` directory is _not_ meant to be shipped in our NPM package.
 // Use e.g. `@import '~carbon-components/scss/globals/scss/styles.scss'` instead.
+
+$css--font-face: false;
+$css--body: false;
+$css--reset: false;
```

Comment:
just wondering, what are these flags for in this specific story?

Reply:
@emyarod IIUC this file is brought in in a separate sass-loader execution context and as a result doesn't share the flag values or imported-modules values 😞 This lead to duplicate css reset rules being set